server/src/main/java/com/lunchtime/models/Restaurant.java

```diff
@@ -52,10 +52,10 @@
     @Column(name = "menu_id")
     private Long menuId;
 
-    @ManyToOne
     @NotNull
+    @ManyToOne(fetch = FetchType.LAZY)
     @JsonIgnore
-    @JoinColumn(name = "person_id")
+    @JoinColumn(name = "person",referencedColumnName = "id")
```

Comment:
I don't understand why, but I can't create restaurant with Person
![image](https://user-images.githubusercontent.com/6152066/77198913-91db8700-6af0-11ea-93c8-976621a62643.png)

Reply:
There are only `id's` in restaurant table in database, so table name `person_id` will be better
![image](https://user-images.githubusercontent.com/6152066/77199038-d535f580-6af0-11ea-8cbd-29b9d4139377.png)

Reply:
To-To 
You want NotBlank, fields must by enter

Reply:
I entered `person` as you can see in screenshot. This request works in dev branch

Reply:
![Screenshot from 2020-03-09 15-35-30](https://user-images.githubusercontent.com/37099996/77200180-ba648080-6af2-11ea-97e0-8d6fcdc33554.png)
Your code)

Reply:
you enter online id)

Reply:
We don't need to send all `Person` entity. It can be inconveniently in fronted, so we can send only id of inner entity.
You can try to do the same request in dev branch. It works
And I definitely didn't write `referevedColumnName = "id"` you can see it in dev branch https://github.com/Social-projects-Rivne/Lunch_Time/blob/dev/server/src/main/java/com/lunchtime/models/Restaurant.java#L56

Reply:
you must have logic in backend(restaurant) to send only one id. For now we have relationship between person and restaurant.

Reply:
you must rewrite your controller(method create). Entity have changing

Reply:
> you must rewrite your controller(method create). Entity have changing
or create task fix

First of all, it isn't my method, it is ours. Every programmer in team has equal rights to every piece of code in project
Second. If you updated something and this changes affect other code and functionality, you should take care about this problem. I even may not know about your changes.

And the last one. You should test your code before PR. Even full `Person` entity doesn't work
![image](https://user-images.githubusercontent.com/6152066/77221169-f679fe80-6b4f-11ea-9413-4dc04ee15d12.png)

Reply:
I fixed this issue, can you checks?

Reply:
Thanks! Previous functionality works well!